concourse/pipelines/create-cloudfoundry.yml

```diff
@@ -1942,6 +1942,23 @@ jobs:
                 - -e
                 - -c
                 - |
+                  # FIXME: Remove this once
+                  # https://github.com/cloudfoundry/cf-acceptance-tests/pull/188 has been merged
```

Comment:
If there is a story for the next upgrade already created we should add a comment about this.